shared/chat/conversation/messages/attachment/image.js.flow

```diff
@@ -7,3 +7,7 @@ export type ImageRenderProps = {
 }
 
 export class ImageRender extends Component<void, ImageRenderProps, void> {}
+
+export function imgMaxWidth() {
+  return 0
```

Comment:
I'm a little confused by this. Why define it like this in the flow file instead of a type definition?